aws/warehouse.go

```diff
@@ -64,6 +67,11 @@ const (
 	backoffMaxElapsedTime      = 10 * time.Second
 )
 
+const (
+	defaultGcFrequency = 1 * 24 * time.Hour /* 1 day */
+	defaultKeysPerAccount = 2 //FIXME my assumption is there is only one key per account
```

Comment:
Does this comment still need to be addressed?

Reply:
Not the keys per account but the GCFrequency. GCP GcFrequency is 14 days, should it be the same for aws?